---
semap_msgs/ObjectDescription[] descriptions
